testhelpers/docker_registry.go

```diff
@@ -120,7 +121,7 @@ func (r *DockerRegistry) Start(t *testing.T) {
 			storageBindPath = "c:/registry-storage"
 		}
 
-		volumeBinds = append(volumeBinds, fmt.Sprintf("%s:%s", r.volumeName, storageBindPath))
+		volumeBinds = append(volumeBinds, net.JoinHostPort(r.volumeName, storageBindPath))
```

Comment:
This is exactly what I saw when I googled for `JoinHostPort`. I thought that I'm missing something but since we both found the same information, I lean toward ignoring the `muse-dev` warning in this case, unless someone else knows something that I didn't find on my small research.